test/FsAutoComplete.Tests.Lsp/TestCases/Tooltips/Script.fsx

```diff
@@ -11,6 +11,6 @@ sprintf "asd"
 /// <param name="b">bar</param>
 /// <param name="a">baz</param>
 /// <returns></returns>
-let someFunction () = ()
+let someFunction (a) (b) (c) = ()
```

Comment:
new FCS defaults to checking xml comments, so this update needed to happen to make the test pass